schema/artwork/index.js

```diff
@@ -39,6 +39,10 @@ import {
   GraphQLInt,
 } from 'graphql';
 
+const is_inquireable = ({ inquireable, acquireable }) => {
+  return (inquireable && !acquireable);
+};
```

Comment:
I think the API always modeled `acquireable` as an additional action that could be taken on certain for-sale works. It was never modeled to be mutually exclusive with inquiries, but in practice designs always presented it instead of inquiry affordances. I.e., the API allows inquiries on e-commerce works, but clients have found it convenient to treat them as mutually exclusive.